forms/MarkdownTextareaField.php

```diff
@@ -34,22 +34,31 @@ class MarkdownTextareaField extends TextareaField
      * @param  array $properties
      * @return string            HTML to be used
      */
-    public function FieldHolder($properties = array())
+    public function FieldHolder($properties = [])
```

Comment:
Hey @matiasnombarasco - this module supports PHP 5.3, so best to leave it the way it was